crates/containerd-shim-wasm/src/testing.rs

```diff
@@ -8,7 +8,11 @@ use std::time::Duration;
 
```

Comment:
Can you update the CHANGELOG for your changes to the containerd-shim-wasm crate?